docker/image/whyis-server-deps/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM tetherlessworld/whyis-ubuntu
+
+RUN puppet module install\
+	puppet-python --version 3.0.1
+RUN puppet module install\
+	puppetlabs-apt --version 7.0.1
+RUN puppet module install\
```

Comment:
Will use the multi-stage build for this later, doesn't need to be in the container.